objects/obj_popup_dialogue/Step_0.gml

```diff
@@ -45,15 +45,19 @@ if (fin=true) and (too_high<=0){fin=false;
                 if (target2="Info Chip") then obj_controller.trade_info-=real(inputing);
             }
             
-            if (target="controller") then obj_controller.cheatcode=real(inputing);
-            
-            if (instance_exists(obj_controller)){obj_controller.cooldown=8;}
-            instance_destroy();
+            if instance_exists(obj_controller)
+                obj_controller.cooldown = 8
```

Comment:
missing brackets or semicolons?

Reply:
please either:

convert to `if instance_exists(obj_controller) then obj_controller.cooldown = 8;`

or

add semicolon: `obj_controller.cooldown = 8;`

or 

add a newline above instance_destroy(); (and add a semicolon)